src/items.zig

```diff
@@ -833,7 +873,10 @@ pub const Recipe = struct { // MARK: Recipe
 };
 
 var arena: main.heap.NeverFailingArenaAllocator = undefined;
-var toolTypes: std.StringHashMap(ToolType) = undefined;
+
+var toolTypeList: ListUnmanaged(ToolType) = undefined;
```

Comment:
Where is this freed?

Reply:
Apparently implicitly by arena free. [Doesn't manual freeing of the resources in arena defy the point of using arena?](http://youtube.com/watch?v=nZNd5FjSquk)

Reply:
Hmm, I see, I really don't like that we cannot see if something is supposed to use the arena or not.
Maybe we should consider using comptime allocators.